contracts/interface/IChainlinkPriceFeedV3.sol

```diff
@@ -13,7 +13,8 @@ interface IChainlinkPriceFeedV3Event {
         NoRoundId,
         InvalidTimestamp,
         NonPositiveAnswer,
-        AnswerIsOutlier
+        AnswerIsOutlier,
+        Timeout
```

Comment:
Where do we use the Timeout enum?

Reply:
oops [f0f8945](https://github.com/perpetual-protocol/perp-oracle-contract/pull/52/commits/f0f89453d12b47de512a038045e1fdba02ba725e)